Libraries/ConfigParser/ConfigParser.Tests/ConfigParserTests.cpp

```diff
@@ -1,113 +1,113 @@
-#include "stdafx.h"

```

Comment:
not sure why these got reformatted as part of this 🤷‍♂️

Reply:
I am not sure either, I did a checkout to origin/master for all of these files but we still get the reformatted issue